### PR TITLE
Add information disable cross-browser support

### PR DESCRIPTION
**Pull request by sahandilshan**
## Purpose
$subject

Screenshot of the on-prem
Screenshot 2023-11-14 at 9 53 47 AM

Screenshot of the asgardeo doc
Screenshot 2023-11-14 at 9 55 36 AM

### CODE REVIEW

File: en/identity-server/next/docs/guides/authentication/passwordless-login/add-passwordless-login-with-magic-link.md [hunk 1]
```diff
@@ -4,4 +4,14 @@
 
 {% set configure_email_sender = "- You need to [configure the email sending module](../../../deploy/configure-email-sending.md) for the WSO2 Identity Server to send the Magic link to the users." %}
 
+{% set disable_cross_browser_support = "If you want to remove the cross-browser functionality, you can do so by adding the following configuration in the `deployment.toml` file.
```
Comment (himeshsiriwardana) [inline en/identity-server/next/docs/guides/authentication/passwordless-login/add-passwordless-login-with-magic-link.md:7]: Please mention the location of the deployment.toml file. Also say what happens with this config.

For example: "Enabling this config forces users to open the magic link from the same browser from which they initiated the login process."

Comment (sahandilshan) [reply en/identity-server/next/docs/guides/authentication/passwordless-login/add-passwordless-login-with-magic-link.md]: Fixed with 5eb3acb1ce64db1871d0c170a1a8e7e2bc61ea99

Screenshot 2023-11-14 at 11 09 06 AM

File: en/identity-server/next/docs/guides/authentication/passwordless-login/add-passwordless-login-with-magic-link.md [hunk 2]
```diff
@@ -4,4 +4,15 @@
 
 {% set configure_email_sender = "- You need to [configure the email sending module](../../../deploy/configure-email-sending.md) for the WSO2 Identity Server to send the Magic link to the users." %}
 
+{% set disable_cross_browser_support = "If you want to remove the cross-browser functionality, you can do so by adding the following configuration in the `<IS_HOME>/repository/conf/deployment.toml` file.
```
Comment (himeshsiriwardana) [inline en/identity-server/next/docs/guides/authentication/passwordless-login/add-passwordless-login-with-magic-link.md:7]:
```suggestion
{% set disable_cross_browser_support = "If you want to remove the cross-browser functionality, you can do so by adding the following configuration in the `deployment.toml` file found in the {IS_HOME}/repository/conf/ directory.
```

File: en/identity-server/next/docs/guides/authentication/passwordless-login/add-passwordless-login-with-magic-link.md [hunk 3]
```diff
@@ -4,4 +4,15 @@
...
+
+        ```toml
+           [session.nonce.cookie]
+           default_whitelist_authenticators=[] 
+        ```
+        This configuration will ensure that the magic link can only be opened in the same browser where the login was originally initiated.
```
Comment (himeshsiriwardana) [inline en/identity-server/next/docs/guides/authentication/passwordless-login/add-passwordless-login-with-magic-link.md:13]:
```suggestion
        This configuration ensures that the magic link can only be opened using the same browser from which the login was initiated.
```

**Review by himeshsiriwardana — APPROVED** [review]

**Pull request merged**